### ISSUE TITLE
关于训练中数据采样的问题？

### ISSUE THREAD

**Issue by Baboom-l** [FoundationVision/GLEE#12]
GLEE在训练过程中，每个iter采样的数据都是来自相同数据集的吗？我发现task总是取batched_inputs[0],

**Comment by wjf5203** [comment]
Yes, since all the images on the same GPU come from the same dataset, taking [0] allows model to know what task the data on this card.
